frontend/src/wasm-communication/messages.ts

```diff
@@ -89,6 +89,7 @@ export class NodeGraphOutput {
 }
 
 export class FrontendNode {
+	[x: string]: any;
```

Comment:
I'm not quite sure what this does?

Reply:
It seems I accidently included that file It doesn't do anything, Sorry.